vendor/bat-native-ads/src/bat/ads/ad_event_history.cc

```diff
@@ -5,6 +5,9 @@
 
 #include "bat/ads/ad_event_history.h"
 
+#include <cstdint>
+
+#include "base/check_op.h"
```

Comment:
Should we include include "base/check.h" for some files? I've found the following comment in check_op.h:

> // This header also provides the (DP)CHECK macros (by including check.h), so if
> // you use e.g. both CHECK_EQ and CHECK, including this header is enough. If you
> // only use CHECK however, please include the smaller check.h instead.
>

Reply:
Fixed